src/pages/en/guides/migrate-to-astro/from-gatsby.mdx

```diff
@@ -2,67 +2,884 @@
 
-## Switch from Gatsby to Astro
 
-To convert a Gatsby blog to Astro, start with our official blog theme starter template, or explore more community blog themes in our [theme showcase](https://astro.build/themes/). 
+## Convert your Project
 
-You can pass a `--template` argument to the `create-astro` command to start a new Astro project from any existing Astro repository on GitHub:
+### Get Started with Astro
 
-  <PackageManagerTabs>
-    <Fragment slot="npm">
-    ```shell
-    npm create astro -- --template withastro/blog
-    ```
-    </Fragment>
-    <Fragment slot="pnpm">
-    ```shell
-    pnpm create astro --template withastro/blog
-    ```
-    </Fragment>
-    <Fragment slot="yarn">
-    ```shell
-    yarn create astro --template withastro/blog
+- Option 1: **Create a new Astro project** 
+
+  Use `npm create astro@latest` to launch Astro's CLI wizard or choose a community theme from the [Astro Theme Showcase](https://astro.build/themes). 
+  
+  
+  You can also pass a `--template` argument to the `create-astro` command to start from an existing Astro repository on GitHub:
+
+    <PackageManagerTabs>
+      <Fragment slot="npm">
+      ```shell
+      npm create astro -- --template <github-username>/<github-repo>
+      ```
+      </Fragment>
+      <Fragment slot="pnpm">
+      ```shell
+      pnpm create astro --template <github-username>/<github-repo>
+      ```
+      </Fragment>
+      <Fragment slot="yarn">
+      ```shell
+      yarn create astro --template <github-username>/<github-repo>
+      ```
+      </Fragment>
+    </PackageManagerTabs>
+  
+  Then, copy your existing Gatsby project files over to your new Astro project into a separate folder outside of `src`.
+
+- Option 2: **Update your existing Gatsby project dependencies**
+
+  Remove any Gatsby-related dependencies in `package.json` and follow all the steps to [install Astro manually](/en/install/manual/) in your existing repository. (You may wish to do this on a different branch.) 
+  
+  Make sure you have [these project files](/en/install/manual/#7-next-steps) and a working home page before continuing. 
+
+:::tip
+Prefer to explore in your browser instead? Visit https://astro.new for a variety of starter templates that can be opened and edited in StackBlitz, CodeSandbox or Gitpod.
+:::
+
+### Install integrations (optional)
+
+You may find it useful to install some of [Astro's optional integrations](/en/guides/integrations-guide/) to use while converting your Gatsby project to Astro:
+
+- **@astrojs/react**: to reuse some existing React UI components in your new Astro site, or keep writing with React components.
+
+- **@astrojs/image**: to replace Gatsby's Image plugin with Astro's own image-optimizing components.
+
+- **@astrojs/mdx**: to bring existing MDX files from your Gatsby project, or to use MDX in your new Astro site.
+
+### Put your code in `src`
+
+1. **Delete** Gatsby's `public/` folder. 
```

Comment:
Only if Option 2 was picket…

Maybe move this to the "what is different" section as a "FYI, this is the difference in the folder tree"